packages/pg/lib/connection.js

```diff
@@ -92,11 +91,17 @@ class Connection extends EventEmitter {
         }
       }
 
-      if (net.isIP(host) === 0) {
-        options.servername = host
+      try {
+        var net = require('net')
+        if (net.isIP && net.isIP(host) === 0) {
+          options.servername = host
+        }
+      } catch (e) {
+        this.emit('error', e)
```

Comment:
Why catch this?

Reply:
There are two approaches to Node.js compatibility in Cloudflare Workers right now.
One uses compile-time shims; the other uses packages built in to the runtime.
The former case, I think, provides an empty object for this import; the latter will throw.
Currently I could not quite get the latter approach working, so I reverted to the former, which probably does not need this try-catch.

I'll see if we can remove it.